<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ButtonGroup" name="btn_group" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="MyFiles" name="page_file">
       <zorder>verticalSpacer</zorder>
      </widget>
      <widget class="ShareList" name="page_share"/>
      <widget class="RankList" name="page_rank"/>
      <widget class="Trans" name="page_trans"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyFiles</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/myfiles.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShareList</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/sharelist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RankList</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/ranklist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Trans</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/trans.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/buttongroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
